### ISSUE TITLE
[Windows]Add windows host, error when performing step 7

### ISSUE THREAD

**Issue by kingsd041** [rancher/rancher#11173]
**Rancher versions:**
rancher/server: v1.6.15-rc2

**Docker version: (`docker version`,`docker info` preferred)**
17.06.2-ee-6
**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**
Windows 2016
**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**
AWS
**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**
Windows
**Steps to Reproduce:**
Add windows host, error when performing step 7

**Results:**
Add failure, The error log:
```
PS C:\Users\Administrator> $obj=$(docker run --rm -v "C:\Program Files\rancher:C:\Program Files\rancher" rancher/agent-w
indows:v0.2.0-rc1 -RegisterUrl http://52.210.162.197:8081/v1/scripts/444C89D7EB91EDE05DBF:1514678400000:CwN77eJeMmSaXDm4
wIorknY -HostLabels "io.rancher.network.per_host_subnet.subnet=192.168.4.0%2F24&io.rancher.network.per_host_subnet.route
r_ip=172.31.250.142")
Remove-Item : Cannot find path 'C:\program files\rancher\download' because it
does not exist.
At C:\program files\run.ps1:111 char:1
+ Remove-Item -Path "$rancherBaseDir/$downloadFolder" -Confirm:$false - ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\program files\rancher\downlo
   ad:String) [Remove-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.Remov
   eItemCommand
```

**Comment by deniseschannon** [comment]
Fix will be in v1.6.15-rc3

**Comment by kingsd041** [comment]
Tested with Rancher v1.6.15-rc3.
In v1.6.15-rc3 version can successfully add the host.
